commands/secret_update.go

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) OpenFaaS Author(s) 2019. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+package commands
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/openfaas/faas-cli/proxy"
+	"github.com/openfaas/faas-cli/schema"
+	"github.com/spf13/cobra"
+)
+
+var (
+	secretValue string
+	secretFile  string
+)
+
+var secretUpdateCmd = &cobra.Command{
+	Use:     "update",
+	Aliases: []string{"replace"},
```

Comment:
I think that one word is better, as having 2 options for 1 thing can be confusing for the user. May be just `u`?